### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by oscarnava**

### CODE REVIEW

**Comment by mosaaleb** [comment]
Reviewing

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,692 @@
...
+  grid-area: icon;
+  width: 0px;
+  height: 0px;
+  margin: 0;
+}
+
+@media screen and (max-width: 1024px) {
+  #whole-nav {
+    display: grid;
+    grid-template-areas: "icon top-nav" "icon bottom-nav";
```
Comment (mosaaleb) [inline styles/styles.css:97]: For more readability, let's put all styles that related to `1024px` breakpoint into the same block, also for `768px`

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Great job @oscarnava  @nicolasmarino99 :clap:. The UI looks very clean. Good use of semantic html tags. You have already handled most of the requirements, yet there's some minor requirements to be met before moving on.

Please see my comments below. 


### Navigation bar
- [x] should use fixed position.

### Latest News section
Screenshot from 2019-07-29 22-15-08

- [x] Each article should have an image on the leftmost part of the article and its title on the right side -- as seen on the TNW page (The text under the title should be aligned with the title)

### Footer
- [ ] The bottom part of the footer should have the TNW logo with a copyright paragraph -- as seen on TNW page.


### General
- [x] Let's add links to anything needs to be a link (articles - articles headers) -- As seen on TNW page.
- [x] The hamburger icon (bars) should not appear on medium nor big screens.


Please make sure that these requirements are done and request a new code review. 

#### Happy Coding :computer:

**Comment by Oghenebrume50** [comment]
Nice guys good one :+1:  everything works.

**Pull request merged**